src/Sarif/ConsoleLogger.cs

```diff
@@ -94,40 +94,47 @@ public void Log(IRule rule, Result result)
 
             WriteToConsole(
                 result.Level,
+                result.Kind,
```

Comment:
>Kind [](start = 23, length = 4)

I suggest putting `kind` before `Level` in the argument list; it's the more general concept.